helm/blueapi/Chart.yaml

```
@@ -17,3 +17,8 @@ type: application
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 # This is updated automatically by CI, the appVersion is automatically inserted by CI
 version: 0.1.0
+
+dependencies:
+  - name: rabbitmq
+    version: 11.16.2
+    repository: oci://registry-1.docker.io/bitnamicharts
```

```suggestion
dependencies:
  - name: rabbitmq
    version: 11.16.2
    repository: oci://registry-1.docker.io/bitnamicharts
    enabled: rabbitmq.enabled
```

If we add this to the conditions, we can also add 
rabbitmq:
  enabled: false

to our values.yaml to disable rabbitmq (once we have BlueAPI not requiring a message bus)